src/docs/recommendation.md

```diff
@@ -26,9 +26,33 @@ G-Rec 2. Provide metadata to describe the crosswalks and individual mappings ([d
 
 #### Technical recommendation
 
-T-Rec 2.1 Use the minimum SSSOM metadata schema defined by the FAIR IMPACT project to document mappings and crosswalks ([discussion#63](https://github.com/mapping-commons/rda-fair-mappings/discussions/63))
-
-T-Rec 2.2 If the mappings are separated from the associated metadata, Include an additional metadata field containing the PID pointing to the mapping or crosswalk  ([discussion#64](https://github.com/mapping-commons/rda-fair-mappings/discussions/64))
+T-Rec 2.1 Use the minimum SSSOM metadata schema defined by the FAIR IMPACT project to document mappings and crosswalks ([discussion#63](https://github.com/mapping-commons/rda-fair-mappings/discussions/63)). We recommend to adopt the following SSSOM metadata elements as mandatory metadata to describe the mappings and crosswalks:
+
+- Metadata elements for **Findability** (SSSOM recommended properties in parentheses):
+   - Author ([author_id](https://mapping-commons.github.io/sssom/author_id/), [author_label](https://mapping-commons.github.io/sssom/author_label/))
+   - Creator ([creator_id](https://mapping-commons.github.io/sssom/creator_id/), [creator_label](https://mapping-commons.github.io/sssom/creator_label/))
+   - Reviewer ([reviewer_id](https://mapping-commons.github.io/sssom/reviewer_id/), [reviewer_label](https://mapping-commons.github.io/sssom/reviewer_label/))
+   - Date of creation of this version ([publication_date](https://mapping-commons.github.io/sssom/publication_date/))
```

Comment:
Is "creation" here 

1. The creation of the mapping set (when was the whole mapping file created)
2. The creation of the individual mapping rule
3. The publication date of the mapping set
?